universal-application-tool-0.0.1/app/controllers/admin/QuestionController.java

```diff
@@ -59,12 +61,12 @@ public QuestionController(
               String exception = "";
```

Comment:
Suggestion (blocking): rename to `errorMessages` since this is not an Exception

Reply:
Done